File: wt_code_challenge_android/app/src/main/java/com/testme/wtcodechallenge/api/model/Ebook.kt
package com.testme.wtcodechallenge.api.model

import com.google.gson.annotations.SerializedName

/**
 * Created by Pablo Reyes [devpab@gmail.com] on 1/11/21.
 */
data class Ebook(
    val title: String?,
    val author: String?,

    @SerializedName("imageURL")
    val image: String?
)

File: wt_code_challenge_ios/wt_code_challenge/Structures/Ebook.swift
//
//  Ebook.swift
//  wt_code_challenge
//
//  Created by Pablo Reyes on 1/11/21.
//

import Foundation

struct Ebook: Decodable, Hashable {
    let title: String?
    let author: String?
    let image: String?
    
    enum CodingKeys: String, CodingKey {
        case title
        case author
        case image = "imageURL"
    }
    
    func hash(into hasher: inout Hasher) {
        hasher.combine(title)
    }
}
